package.json

```diff
@@ -30,10 +30,14 @@
     }
   ],
   "type": "module",
-  "exports": "./dist/index.mjs",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.cjs"
+  },
   "main": "dist/index.mjs",
```

Comment:
This is still a breaking change for pre-exports node versions. Your "main" must be a `.js` file that's CJS to avoid that.